Doc/library/importlib.rst

```diff
@@ -1284,66 +1288,71 @@ find and load modules.
    A specification for a module's import-system-related state.  This is
    typically exposed as the module's ``__spec__`` attribute.  In the
    descriptions below, the names in parentheses give the corresponding
-   attribute available directly on the module object.
-   E.g. ``module.__spec__.origin == module.__file__``.  Note however that
-   while the *values* are usually equivalent, they can differ since there is
-   no synchronization between the two objects.  Thus it is possible to update
-   the module's ``__path__`` at runtime, and this will not be automatically
-   reflected in ``__spec__.submodule_search_locations``.
+   attribute available directly on the module object. For example,
+   ``module.__spec__.name == module.__name__``.  Note however that while
+   the *values* are usually equivalent, they can differ since there is no
+   synchronization between the two objects.  For example, it is possible
+   to update the module's ``__name__`` at runtime and this will not be
+   automatically reflected in the module's ``__spec__.name``, or the
+   other way round.
 
    .. versionadded:: 3.4
 
    .. attribute:: name
 
    (``__name__``)
 
-   A string for the fully-qualified name of the module.
+   The module's fully-qualified name.
 
    .. attribute:: loader
 
    (``__loader__``)
 
-   The loader to use for loading.  For namespace packages this should be
-   set to ``None``.
+   The module's loader.
```

Comment:
I agree with the first sentence. To be consistent with your suggestion above, we could rephrase it:

> The loader to use when importing the module.

I disagree with the second sentence. As I reported on the description of this PR, on [Stackoverflow](https://stackoverflow.com/questions/52869541) and in the [Python Bug Tracker](https://bugs.python.org/issue35181), both `__loader__` and `__spec__.loader` attributes of a namespace package are not `None` but the same `_frozen_importlib_external._NamespaceLoader` instance, since [Barry Warsaw's PR](https://github.com/python/cpython/pull/5481) was merged on February 2018.

Reply:
re-visited in a follow-up review